src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's Block number in the address book.
+ */
+
+public class Block {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.